diff -uNr mopac7-1.13.orig/configure mopac7-1.13/configure
--- mopac7-1.13.orig/configure	2007-09-21 16:58:57.000000000 +0000
+++ mopac7-1.13/configure	2007-10-25 01:04:18.000000000 +0000
@@ -18990,7 +18990,6 @@
 # If libf2c is present, it is used and both the library and the executable is built.
 # If only libg2c is present, then only the library is built and the executable is not built.
 saved_flags=$CFLAGS
-CFLAGS="-Dmain=MAIN__"
 have_f2c="no"
 { echo "$as_me:$LINENO: checking for main in -lf2c" >&5
 echo $ECHO_N "checking for main in -lf2c... $ECHO_C" >&6; }
diff -uNr mopac7-1.13.orig/configure.ac mopac7-1.13/configure.ac
--- mopac7-1.13.orig/configure.ac	2007-09-21 16:58:50.000000000 +0000
+++ mopac7-1.13/configure.ac	2007-10-25 01:03:57.000000000 +0000
@@ -50,7 +50,6 @@
 # If libf2c is present, it is used and both the library and the executable is built.
 # If only libg2c is present, then only the library is built and the executable is not built.
 saved_flags=$CFLAGS
-CFLAGS="-Dmain=MAIN__"
 have_f2c="no"
 AC_CHECK_LIB([f2c], [main], [have_f2c="yes"; CFLAGS=$saved_flags],
 	[# If not found, try -lg2c:
